### PR TITLE
Revamp of home page and other dev workflow optimizations 

### PR DESCRIPTION
**Pull request by devroy10**
[video_of_new_home_page_desktop.webm](https://github.com/user-attachments/assets/ca8a2e69-7f5c-4a84-95f0-580f66e678f8)

[video_of_new_home_page_mobile.webm](https://github.com/user-attachments/assets/a2cd00a0-a46f-494f-aefb-405ecc961a7c)

Blank positions on the hero section and features section are for images not yet included in the repo.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,9 +1,28 @@
-@fontsource/lexend;
```
Comment (stawuah) [inline src/App.css:1]: @devroy10  kindly use `lexend` as our custom font  👌🏾

File: src/components/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          {bannerCopy}
+        </h1>
+
+        <CtaButton
+          linkTo="/get-started"
+          title="Get Started Now"
+          otherClasses="bg-white w-max font-medium border border-[#E3E3E3] "
+        />
+
+        {/* <div className='aspect-square bottom-[-230px] flex-none h-[var(auto,600px)] left-[-230px] overflow-hidden pointer-events-none absolute w-[600px] z-[1]  hidden md:block'>
```
Comment (stawuah) [inline src/components/Banner.tsx:19]: @devroy10  , kindly clean up commented or unused code in the codebase

File: src/components/ui/CtaButton.tsx
Comment (stawuah) [inline src/components/ui/CtaButton.tsx:1]: @devroy10  , `CtaButton` file names should be understandable by fellow devs  who might be refactoring

File: src/index.css [hunk 3]
```diff
@@ -1,4 +1,10 @@
+/* montserrat font import  */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap');
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-  
\ No newline at end of file
+
+body {
```
Comment (stawuah) [inline src/index.css:8]: @devroy10, kindly add `lexend ` to this part

File: src/pages/UserForm.tsx
Comment (stawuah) [inline src/pages/UserForm.tsx:1]: @devroy10, Can you elaborate on what this form does ?

File: tailwind.config.js [hunk 4]
```diff
@@ -37,7 +39,25 @@ module.exports = {
     extend: {
       fontFamily: {
         sans: ['Source Sans Pro', 'sans-serif'],
-        lexend: ['Lexend', 'sans-serif'],
+        // lexend: ['Lexend', 'sans-serif'],
```
Comment (stawuah) [inline tailwind.config.js:42]: Lexend  here 😌

File: tailwind.config.js
Comment (stawuah) [inline tailwind.config.js:1]: LGTM 😌,  @devroy10  you should be much interested in  the comments i added 




THANKS 🌚.

**Review by stawuah — APPROVED** [review]
looks good to @Emmilija and I, also make changes based on the comments on the review

**Comment by devroy10** [comment]
Alright, I will make the changes.

File: src/components/Banner.tsx (hunk 2 above)
Comment (devroy10) [reply src/components/Banner.tsx:19]: this code is actually meant for assets depicting two real estate peoperties, positioned at the bottom edges of the banner.
I would be used when the assets are provided

File: src/pages/UserForm.tsx
Comment (devroy10) [reply src/pages/UserForm.tsx:1]: I didn't create this form component, but I will look into it.

**Pull request merged**